tests/SConscript

```diff
@@ -2,4 +2,5 @@
 import os
 from lsst.sconsUtils import scripts, env
 
-scripts.BasicSConscript.tests()
+# testCoadds.py must be entirely skipped, and Scons doesn't like that
+scripts.BasicSConscript.tests(ignoreList=['testCoadds.py'])
```

Comment:
Please supplement this code comment with some explanation of why testCoadds.py can no longer be run.